src/osvr/Server/CMakeLists.txt

```diff
@@ -18,6 +19,8 @@ set(SOURCE
     ServerImpl.h
     "${CMAKE_CURRENT_BINARY_DIR}/display_json.h")
 
+configure_file(ConfigFilePaths.h.cmake_in "${CMAKE_CURRENT_BINARY_DIR}/ConfigFilePaths.h")
```

Comment:
I don't think this file needs to be generated - it doesn't look like it's using any CMake substitutions. Just consuming `#ifdefs` from a configured header is fine in a regular header or implementation file.